### ISSUE TITLE
Training model with a dataset

### ISSUE THREAD

**Issue by gcunhase** [fatchord/WaveRNN#13]
Any hints on how to use NB2 to train a dataset (say a directory with multiple audio files) and then use the trained model to generate one of those samples?

Thank you in advance

**Comment by fatchord** [comment]
@gcunhase Yeah it should be easy enough - you could try Pytorch's Dataset class. Whether the model will produce one of the samples is a gamble since the model would be unconditioned and you wouldn't have much, if any, control over what it outputs.

**Comment by gcunhase** [comment]
@fatchord Thank you for your reply, would it be possible to condition it somehow?

**Comment by fatchord** [comment]
@ Have a look at notebooks 4a and 4b - there you'll see how I do it.

**Comment by hyzhan** [comment]
Is that have some sample audio？ I am trying to train on the LJspeech dataset for seq_len=8000,batch=12,learning_rate=1e-3 warm up 4k step and then decay, my loss is around 1.7

**Comment by fatchord** [comment]
@hyzhan what model are you training? By the way the seq_len should be around 1000 for more efficient training - also make sure it's cleanly divisible by your sprectrogram hop length.

**Comment by hyzhan** [comment]
Thanks for reply, I have successfully trained in the alternative model, it sounds good, thank you very much. But the strange thing is that the alternative model can converge in a dataset of about 5 hours, but the dataset is often jittery in 30 hours, causing the latter to generate audio with loud noise.
![30hour_train_loss](https://user-images.githubusercontent.com/14198831/47264902-ea040780-d551-11e8-8794-0481ebfc0785.png)
![30hour_train_loss2](https://user-images.githubusercontent.com/14198831/47264903-eec8bb80-d551-11e8-95eb-0c8a23fb4225.png)
Above are two large data sets, the smaller ones below: 
![5hour_train_loss](https://user-images.githubusercontent.com/14198831/47264904-f25c4280-d551-11e8-9e40-0f89a963c352.png)
Could you please give me some advice?
